spec/tests/ContentMetadata.spec.ts

```diff
@@ -43,6 +43,54 @@ describe('content metadata spec', () => {
       }));
     });
 
+    it('default streamType in custom tags', () => {
+      jest.spyOn(playerMock, 'getStreamType').mockReturnValue(StreamType.Dash);
+      playerMock.eventEmitter.firePlayEvent();
+
+      expect(convivaVideoAnalytics.reportPlaybackRequested).toHaveBeenLastCalledWith(expect.objectContaining({
+        streamType: 'dash',
+      }));
+    });
+
+    it('default vrContentType in custom tags', () => {
+      playerMock.eventEmitter.firePlayEvent();
+
+      expect(convivaVideoAnalytics.reportPlaybackRequested).toHaveBeenLastCalledWith(expect.objectContaining({
+        vrContentType: undefined,
+      }));
+    });
+
+    it('override playerType in custom tags', () => {
+      jest.spyOn(playerMock, 'getPlayerType').mockReturnValue(PlayerType.Native);
+      jest.spyOn(playerMock, 'getStreamType').mockReturnValue(StreamType.Dash);
+      convivaAnalytics.updateContentMetadata({ custom: { myTag: 'withMyValue', playerType: PlayerType.Html5 }, assetName: 'MyAsset' });
```

Comment:
Nit: Not 100% related to this PR, but might be worth adding a test case where we check that `myTag` also ends up in the object `reportPlaybackRequested` is called with.

Reply:
A test case for custom tags already exists.

```
describe('externally content metadata updates', () => {
   ......................................................
   ..................................................... 
   it('custom tags', () => {
```

Reply:
I search for this but still missed it 🤦 Thanks for pointing me to it!